packages/polling-controller/src/StaticIntervalPollingController.ts

```diff
@@ -30,8 +30,18 @@ function StaticIntervalPollingControllerMixin<
   {
     readonly #intervalIds: Record<PollingTokenSetId, NodeJS.Timeout> = {};
 
+    #durationIds: Record<PollingTokenSetId, number> = {};
+
     #intervalLength: number | undefined = 1000;
 
+    setKeyDuration(key: string, duration: number) {
```

Comment:
```suggestion
    setDurationForId(id: string, duration: number) {
```

Just to keep in line with the `id` terminology of `PollingTokenSetId`, which is what your keys are.